### ISSUE TITLE
Message of the day is overwritten when OTOBO is upgraded

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#1281]
It looks like the standard way of setting a MOTD is to edit _Kernel/Output/HTML/Templates/Standard/Motd.tt_ . But, when running under Docker, that file is probably overwritten when OTOBO is updated.

**Comment by stefanhaerter** [comment]
Would the solution here be to use a `Custom/` file as well?

**Comment by bschmalhofer** [comment]
Yes, the content of the MOTD file is handled by `$LayoutModule->Output()` which respects the custom template dir.

TODO:

- [ ] Test the custom template dir for MOTD
- [ ] Update the documentation

**Comment by stefanhaerter** [comment]
Putting this in the issue project.

**Comment by svenoe** [comment]
This should just be handled via the Custom directory if used.
